docs/designs/025-processing-startegy-for-health-checks.md

```
@@ -0,0 +1,639 @@
+  BehaviourOverrides drainOverrides = 15;
+  
+  // NEW: Client-requested processing strategy.
+  // STORE_ONLY means the event is for observability only - no cluster resources should be modified.
+  ProcessingStrategy processingStrategy = 16;
+}
+```
+---
+
+### Step 2: Health Monitor Implementation
+
+**Syslog health monitor**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)